example.yaml

```diff
@@ -228,6 +227,19 @@ signalfx_per_tag_api_keys:
   - name: "asf"
     api_key: "definitely_no_farts"
 
+# == X-Ray ==
+# X-Ray can be a sink for trace spans.
+
+# If present, X-Ray will be enabled as a tracing sink
```

Comment:
I'd suggest "If present _and_  `xray_sample_percentage` is non-zero" (and updating the logic for determining if it's enabled accordingly).